Makefile

```diff
@@ -115,10 +116,14 @@ get-version:
 ##############################################################################
 
 .PHONY: docs
-docs:
+docs: tf-docs
 	@rm -rf build/html_docs
 	sh script/build_apm_docs.sh apm-server docs/index.asciidoc build
 
+.PHONY: tf-docs
+tf-docs: $(TERRAFORMDOCS)
+	@$(foreach dir, $(wildcard infra/terraform/modules/*),terraform-docs markdown --hide-empty --header-from header.md --output-file=README.md --output-mode replace $(dir);)
```

Comment:
```suggestion
	@$(foreach dir, $(wildcard infra/terraform/modules/*),$(TERRAFORMDOCS) markdown --hide-empty --header-from header.md --output-file=README.md --output-mode replace $(dir);)
```

I think?

Reply:
Maybe use a pattern rule?

Reply:
fixed and added a pattern rule to make it more "make"